package.json

```diff
@@ -12,7 +12,7 @@
     "url": "https://github.com/atom/atom/issues"
   },
   "license": "MIT",
-  "electronVersion": "2.0.18",
+  "electronVersion": "3.1.8",
```

Comment:
@rafeca 
Electron 3.1.9 is also available with some small fixes: https://github.com/electron/electron/releases/tag/v3.1.9

Reply:
@mfonville: Thanks! Updated in b2a4810eaf0d2f07acff056e03d6159030c0da0d.